python-backend/app/api/incidents/models/mine_incident.py

```diff
@@ -82,12 +84,16 @@ class MineIncident(AuditMixin, Base):
                                                   lazy='joined',
                                                   uselist=False)
 
-    recommendations = db.relationship('MineIncidentRecommendation', lazy='selectin')
+    _recommendations = db.relationship('MineIncidentRecommendation', lazy='dynamic')
     documents = db.relationship('MineIncidentDocumentXref', lazy='joined')
     incident_documents = db.relationship('MineDocument',
                                          lazy='joined',
                                          secondary='mine_incident_document_xref')
 
+    @hybrid_property
```

Comment:
Does this cause this property to be applied to all recomendations returned from queries on this model?